src/tracks/InstrumentTrack.cpp

```diff
@@ -1148,6 +1174,29 @@ class fxLineLcdSpinBox : public LcdSpinBox
 			gui->fxMixerView()->setFocus();// set focus to fxMixer window
 			//engine::getFxMixerView()->raise();
 		}
+
+		virtual void contextMenuEvent( QContextMenuEvent* event )
+		{
+			// for the case, the user clicked right while pressing left mouse-
+			// button, the context-menu appears while mouse-cursor is still hidden
+			// and it isn't shown again until user does something which causes
+			// an QApplication::restoreOverrideCursor()-call...
+			mouseReleaseEvent( NULL );
```

Comment:
This logic was ported from the generic `FxSpinBox` class, but I personally don't know if this can be the case.